client/src/dashboard/RewritingRuleModal.js

```diff
@@ -223,6 +311,7 @@ const RewritingRuleModal = NiceModal.create(({user_id, rule=null, query=null}) =
     //        1: {count: 30, added: true, removed: undefined, value: "this is the string being added"},
     //        2: {count: 33, added: undefined, removed: undefined, value: "this is the string without change"}]
     const diffs = diffChars(q0Line, q1Line);
+    console.log(diffs);
```

Comment:
If we want to keep this console output, let's make it more informative. For example, console.log('[DEBUG] getDiffByLine:');
console.log(diffs);

Reply:
Thank you so much! Fixed.